lib/Frontend/FrontendInputsAndOutputs.cpp

```diff
@@ -1,4 +1,4 @@
-//===--- FrontendInputs.cpp -------------------------------------*- C++ -*-===//
+//===--- FrontendInputsAndOutputs.cpp ----------------------------*-C++ -*-===//
```

Comment:
I think the space needs to be here to be recognized as an emacs-style file type marker...but you actually don't need it at all for .cpp files. (Look around at how other files do it.)

Reply:
Ah, thanks. I checked and fixed all the .cpp files in Frontend, which IIRC is the only place I added .cpp files.